autosarfactory/autosar_ui.py

```diff
@@ -2,22 +2,47 @@
 import tkinter as tk
 import tkinter.ttk as ttk
 import tkinter.font as tkFont
-from ttkthemes import ThemedStyle
+import re
 import os,itertools
-from .autosarfactory import AutosarNode, Referrable
+from ttkthemes import ThemedStyle
+from .autosarfactory import Referrable
+from tkinter import Menu
 
 __resourcesDir__ = os.path.join(os.path.dirname(__file__), 'resources')
 __PAD_X__ = 5 # For some additional padding in the column width
 
 class Application(tk.Frame):
+
+    def selectTheme(self, themeName):

```

Comment:
- I think, its better to keep __init__ ctor as the first method of the class. 
- Also, if the select theme is just used within the class, then it makes sense to put 2 underscores(__) in front of the method to make it private. Normally, this is the standard of making a method private.For protected method, we have to use one underscore(_). 

This is just for readability purpose. But, in reality, there doesn't exist any private/protected concept in python as python treat all methods as symbols.

Reply:
Done.